lib/runFromNetwork.dart

```diff
@@ -40,7 +44,9 @@ class RunFromNetwork extends StatefulWidget {
     this.downloadHash,
     this.downloadByteCodeImage,
     this.downloadDebugInfo,
+    this.errorBuilder,
     this.customNamespaces,
+    this.debugMode = false,
```

Comment:
I just noticed this is failing out the debug mode tests. Should this instead be default initialized to `kDebugMode` if the user doesn't specify?

Reply:
Thats a very good idea, I added that in the new commit